HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,706 @@
+3. As an hts-token-sender, I understand my call will fail if I attempt to send an HTS token using the CryptoTransfer transaction to a receiver who has not associated  with this token and has no open auto-association slot.
+4. As an hts-token-sender I want to cover the custom fees (fixed, fractional, and royalty excluding fallback) of a transfer to an account that is already associated with a token to reduce the friction by a receiving account
+5. As an hts-token-sender I want to cover the fees (association & rent until the end of the first full auto renewal period) of a transfer that consumes an automatic token slot to reduce the friction by a receiving account
+
+#### Receiver
+
+1. As a receiver with no open auto association slots, I don’t want to receive any tokens from a sender using CryptoTransfer to transfer tokens which I have not previously associated with.
+2. As a receiver with open auto association slots, I want to automatically receive tokens from a sender using CryptoTransfers to transfer tokens I have not previously associated with. 
+3. As a user with receiver signature required enabled any airdrops sent to me will fail regardless of open slots.
+
+### Token Airdrop User Stories
+
+#### Sender
+
+1. As an hts-token-airdrop-sender, I want to airdrop a token to a receiver who has already associated with the token. I expect this should work exactly like the crypto transfer.
+2. As an hts-token-airdrop-sender,  I want to send token using to a receiver who has not associated with the token but has an open autoAssociation slot. I expect this should work exactly like the cryptoTransfer.
+3. As an hts-token-airdrop-sender,  I want to send token to a receiver who has not associated with the token and has no open autoAssociation slots. I expect this transfer will be marked as pending that the receiver can claim at any time if it is still available at that time.
+4. As an hts-token-airdrop-sender I want to cover the custom fees (fixed, fractional, and royalty excluding fallback) of an airdrop to an account that is already associated with a token to reduce the friction by a receiving account
+5. As an hts-token-airdrop-sender I want to cover the fees (custom, association & rent until the end of the first full auto renewal period) of an airdrop that consumes an automatic token slot to reduce the friction for a receiving account
+6. As an hts-token-airdrop-sender I want to cover the fees (airdrop, custom, association & rent until the end of the first full auto renewal period) of an airdrop that results in a pending transfer to reduce the friction by the intended receiving account.
+7. As an airdrop-sender I want to cancel pending token transfers that I initiated and are still unclaimed by the receiver.
+8. As a token treasury account holder, I want to airdrop tokens with custom fallback fees
```

Comment:
Isn't this backwards?

Reply:
Not necessarily.
Treasury accounts are not assessed custom fees so as a result they are the only ones that could airdrop a token with a custom fallback fee.
Other accounts that attempt it would require custom fee assessment, if there is a fallback fee in that case the transaction would fail

Reply:
i was being elliptical, sorry.  I mean - this HIP explicitly says twice that you can't at all airdrop tokens with custom fallback fees, it's not supported, right?

Reply:
Correct. It isn't supported because normal account will see a failed transaction and treasury accounts of a token will not be assessed the custom fees.
In both case no fallback fee is ever assessed a receiver.
I've update the user story